images/router/haproxy/reload-haproxy

```diff
@@ -1,21 +1,64 @@
 #!/bin/bash -xu
 
+script=$(basename "${BASH_SOURCE[0]}")
+DEFAULT_COALESCE_INTERVAL=15   #  default 15 secs.
+COALESCE_MAX_WAIT=$((5 * 60))  #  5 minutes
```

Comment:
Could use either  - just added it to indicate a constant (though that said all the other older stuff below is in lower case). So if you want me to, I can change it to lower case. 
`BASH_SOURCE` however needs to be uppercase - its the script/source name (from bash).